stories.json

```diff
@@ -31,7 +31,97 @@
+  },
+  {
+    "permalink": "afk-for-everything",
+    "text": "AFK channel includes notes about family illness, going to the gym, taking a walk, and cooking lunch. To name just a few. People are encouraged to have a life outside of work.",
+    "background": "northernlights"
+  },
+  {
+    "permalink": "sharing-is-caring",
+    "text": "Communication can be formal or informal, it can be oral or written. I love our chat- insert topic people are interested (eg politics, music, pets, parenting, sports, book etc) channels. Why? Because people with less business touch points talk to each other, because more introverted people share, because you chose about what you want to talk <3 As a plus: We also have the channel #YELLING and #metalistening. For me this shows very good, what safe space Giant Swarm is for working.",
+    "background": "northernlights"
+  },
+  {
+    "permalink": "sustainability",
+    "text": "Beyond claiming that we care for the environment, we actively offset all flight emissions that are a result of company-related travel.",
+    "background": "northernlights"
+  },
+  {
+    "permalink": "leadership",
+    "text": "The definition of a manager is a barrier removal professional and above that we, the founders, were always like that, giving people freedom and allowing them to do work. We don't like to 'give' people tasks.",
+    "background": "northernlights"
+  },
+  {
+    "permalink": "safe-place-to-share",
+    "text": "In Slack, somebody transparently said they were suffering from imposter syndrome as everybody else always seemed so productive / good at what they did and they didn't feel the same way. The reactions were so overwhelming and kind, it taught me a lot about the culture even before I started.",
+    "background": "northernlights"
+  },
+  {
+    "permalink": "away-from-keyboard",
+    "text": "In the AFK Slack channel, people openly state all kinds of random reasons why they are not at their keyboard for a while. This has really helped me to understand how 'flexible work hours' and 'self-responsibility' were lived at GS. Furthermore, it is a sign for trust because false excuses or secret AFKs seem to be way less likely to occur.",
+    "background": "northernlights"
+  },
+  {
+    "permalink": "fair-salaries",
+    "text": "Giant Swarm might be the only company who never asked me for my salary expectations. Or worse, try to extract what i was getting paid before in order to not risk “overpaying” me. What this means to me is giant swarm is not a company that takes advantage of people who dont negotiate or people who started at or are coming from low salary positions. Every company says people are their most important assets. But right from the start, this is disproven when the relationship starts with the company taking advantage of their advantageous bargaining position and allowing salaries to wildly converge not because of value but because of employees’ lack of market information.",
```

Comment:
```suggestion
    "text": "Giant Swarm might be the only company who never asked me for my salary expectations. Or worse, try to extract what I was getting paid before in order to not risk “overpaying” me. What this means to me is Giant Swarm is not a company that takes advantage of people who don't negotiate or people who started at or are coming from low salary positions. Every company says people are their most important assets. But right from the start, this is disproven when the relationship starts with the company taking advantage of their bargaining position and allowing salaries to wildly converge not because of value but because of employees’ lack of market information.",
```